src/coreclr/nativeaot/Runtime/i386/AsmMacros.inc

```diff
@@ -165,6 +158,8 @@ RhpWaitForGC2                               equ @RhpWaitForGC2@4
 RhpTrapThreads                              equ _RhpTrapThreads
 RhpStressGc                                 equ @RhpStressGc@0
 RhpGcPoll2                                  equ @RhpGcPoll2@4
+RhHandleGet                                 equ @RhHandleGet@4
+RhpGcSafeZeroMemory                         equ @RhpGcSafeZeroMemory@8
```

Comment:
These fake references needs comments. I am wondering whether it would be better to add these fake references in startup.cpp instead.

Reply:
It's somewhat more verbose to write this in C but I am fine either way. I'll add a comment.